src/api/v1/community_businesses/get.ts

```diff
@@ -24,8 +25,18 @@ export default [
       },
       response: { schema: response },
     },
-    handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit) => {
-      return Boom.notFound('Not implemented');
+    handler: async (request: GetCommunityBusinessesRequest, h: Hapi.ResponseToolkit) => {
+      const {
+        auth: { credentials: { role } } ,
+        server: { app: { knex } },
+        query,
+      } = request;
+
+      if (role !== RoleEnum.TWINE_ADMIN) {
```

Comment:
Is this necessary? No other role (except `FUNDING_BODY_ADMIN`, which doesn't exist yet) has the `org_details-child` scope.

Reply:
I think so. I want to ensure that we do not forget to add functionality for `FUNDING_BODY_ADMIN`.

Reply:
If you just use the `isChildOrg` pre-req this is done for you.